dbhydra/dbhydra_core.py

```diff
@@ -497,11 +496,15 @@ def generate_table_dict(self):
 
 
 class BigQueryDb(AbstractDB):
-    def __init__(self, credentials_path, project_id, dataset_name):
+    def __init__(self, db_details):
         # super().__init__(None,None)
-        self.credentials_path = credentials_path
-        self.project_id = project_id
-        self.dataset = dataset_name
+        self.credentials_path = db_details["DB_SERVER"]
```

Comment:
This seems unsafe. What if db_details do not have "DB_SERVER" key? Use db_details.get or some assert before

Reply:
This is never checked in dbhydra, if you look at class AbstractDB for example.